.github/workflows/benchmarks.yml

```diff
@@ -38,6 +41,26 @@ jobs:
         echo "Waiting for instance to be ready..."
         aws ec2 wait instance-running --instance-ids ${INSTANCE_ID}
 
+  eslint:
```

Comment:
I think it's better if we move the eslint job to a separate workflow, then we won't need to run this heavy workflow on changes to `site`

Reply:
Makes sense. I'll work on that 👍 .